### PR TITLE
[CBRD-24502] manual for not updating statistics when performing DDL

### PR DESCRIPTION
**Pull request by shparkcubrid**
http://jira.cubrid.org/browse/CUBRIDMAN-144
http://jira.cubrid.org/browse/CBRD-24502

manual for not updating statistics when performing DDL

### CODE REVIEW

File: ko/sql/tuning.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@
   
     UPDATE STATISTICS ON CATALOG CLASSES [WITH FULLSCAN]; 
 
-*   **WITH FULLSCAN**: 지정된 테이블의 전체 데이터를 가지고 통계 정보를 업데이트한다. 생략 시 샘플링한 데이터를 가지고 통계 정보를 업데이트한다. 대부분 통계 정보 갱신은 샘플링 정보를 업데이트하는 것으로 충분하며, **WITH FULLSCAN** 은 시스템에 부담을 줄 수 있으므로 가급적 사용을 자제할 것을 권장한다. 
+*   **WITH FULLSCAN**: 지정된 테이블의 전체 데이터를 가지고 통계 정보를 업데이트한다. 생략 시 샘플링한 데이터를 가지고 통계 정보를 업데이트한다. 샘플링한 데이터의 개수는 테이블의 전체 페이지 수와 관계없이 7개이다.
```
Comment (beyondykk9) [inline ko/sql/tuning.rst]:
```suggestion
*   **WITH FULLSCAN**: 지정된 테이블의 전체 데이터를 가지고 통계 정보를 업데이트한다. 생략 시 샘플링한 데이터를 가지고 통계 정보를 업데이트한다. 데이터 샘플은 테이블 전체 페이지 수와 상관없이 7페이지이다.
```

**Review by beyondykk9 — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

File: en/sql/tuning.rst [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
+Statistics for tables and indexes enables queries of the database system to process efficiently. Statistics are not updated automatically for DDL statements such as **CREATE INDEX**, **CREATE TABLE** and DML statements such as **INSERT** and **DELETE**. **UPDATE STATISTICS** statement is the only way to update statistics. So it is necessary to update the statistics by **UPDATE STATISTICS** statement(See :ref:`info-stats`).
 
-**UPDATE STATISTICS** statement is recommended only when a mass of **INSERT** or **DELETE** statements make the big difference between the statistics and the actual information.
+**UPDATE STATISTICS** statement is basically recommended to be executed periodically. It is also recommended to execute when a new index is added or when a mass of **INSERT** or **DELETE** statements make the big difference between the statistics and the actual information.
```
Comment (hgryoo) [inline en/sql/tuning.rst]: How about removing `basically`? This sentence is already complete without the word.

**Review by hgryoo — APPROVED** [review]

File: en/sql/tuning.rst (hunk 2 above)
Comment (shparkcubrid) [reply en/sql/tuning.rst]: Okay. I agree.

Comment (shparkcubrid) [inline en/sql/tuning.rst]:
```suggestion
**UPDATE STATISTICS** statement is recommended to be executed periodically. It is also recommended to execute when a new index is added or when a mass of **INSERT** or **DELETE** statements make the big difference between the statistics and the actual information.
```

File: ko/sql/tuning.rst [hunk 3]
```diff
@@ -6,9 +6,9 @@
 통계 정보 갱신
 ==============
 
-테이블과 인덱스에 대한 통계 정보는 데이터베이스 시스템이 질의를 효과적으로 처리할 수 있게 한다. 통계 정보는 테이블의 생성, 인덱스의 생성/삭제 등 DDL 문이 수행되면 자동으로 갱신된다. 그러나, INSERT, DELETE 등 DML 문이 수행되면 자동으로 갱신되지 않으므로 필요한 경우 사용자가 직접 **UPDATE STATISTICS** 문을 수행하여 통계 정보를 갱신해야 한다(:ref:`info-stats` 참고)
+테이블과 인덱스에 대한 통계 정보는 데이터베이스 시스템이 질의를 효과적으로 처리할 수 있게 한다. 통계 정보는 인덱스의 생성, 테이블의 생성등 DDL 문과 INSERT, DELETE 등 DML 문에 대해서 자동으로 갱신되지 않는다. **UPDATE STATISTICS** 문만이 통계정보를 갱신하는 유일한 방법이다. 필요한 경우 사용자가 직접 **UPDATE STATISTICS** 문을 수행하여 통계 정보를 갱신해야 한다(:ref:`info-stats` 참고)
 
-**UPDATE STATISTICS** 문은 대량의 **INSERT**, 혹은 **DELETE** 문이 수행되어 실제 정보와 통계 정보 사이에 차이가 커질 때 수행할 것을 권장한다.
+**UPDATE STATISTICS** 문은 기본적으로 주기적으로 수행할 것을 권장한다. 또한 새로운 인덱스가 추가되거나, 대량의 **INSERT**, 혹은 **DELETE** 문이 수행되어 실제 정보와 통계 정보 사이에 차이가 커질 때 수행할 것을 권장한다.
```
Comment (shparkcubrid) [inline ko/sql/tuning.rst]:
```suggestion
**UPDATE STATISTICS** 문은 주기적으로 수행할 것을 권장한다. 또한 새로운 인덱스가 추가되거나, 대량의 **INSERT**, 혹은 **DELETE** 문이 수행되어 실제 정보와 통계 정보 사이에 차이가 커질 때 수행할 것을 권장한다.
```

**Review by hyunikn — APPROVED** [review]

**Review by ctshim — APPROVED** [review]

**Pull request merged**